unit/solvers/smt2_incremental/convert_expr_to_smt.cpp

```diff
@@ -714,3 +714,44 @@ SCENARIO(
     }
   }
 }
+
+SCENARIO(
+  "Logical Right-shift expressions are converted to SMT terms",
+  "[core][smt2_incremental]")
+{
+  GIVEN("An integer bitvector")
+  {
+    const auto one_bvint = from_integer({1}, signedbv_typet{8});
+
+    WHEN("We construct a lshr_exprt and convert it to an SMT term")
+    {
+      const auto shift_expr = lshr_exprt{one_bvint, one_bvint};
```

Comment:
We should use two different operands, so that the test checks that the operands don't get mixed up in the conversion. This comment also applies to places with the :dash: emoji.